lib/crypto/packaged.py

```diff
@@ -20,11 +18,11 @@ class SigningError(Exception):
     pass
 
 
-def get_endpoint(file_obj):
-    """Get the endpoint to sign the file, depending on its review status."""
-    server = settings.SIGNING_SERVER
-    if file_obj.version.addon.status != amo.STATUS_PUBLIC:
-        server = settings.PRELIMINARY_SIGNING_SERVER
+def get_endpoint(full=False):
+    """Get the endpoint to sign the file, either the full or prelim one."""
+    server = settings.PRELIMINARY_SIGNING_SERVER
+    if full:
+        server = settings.SIGNING_SERVER
```

Comment:
If `full` is only used to determine which server to use, why not just pass `server` instead, defaulting to the full one? I think `sign_file(obj, server=settings.PRELIMINARY_SIGNING_SERVER)` is a lot clearer as to what will happen than passing `full=True`.